.github/workflows/deploy-function-app.yml

```
@@ -49,7 +53,7 @@ jobs:
       - name: Setup .NET
         uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: 8.0.x
+          dotnet-version: 9.0.x
           
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)